### PR TITLE
CASMPET 6721: Upgrade Istio to 1.19.10

### PR DESCRIPTION
**Pull request by arka-pramanik-hpe**
## Summary and Scope

Upgrade Istio to 1.19.10
Updated the charts from open source to include the necessary changes.
Bumped the minor versions of cray-istio, cray-istio-deploy, and cray-istio-operator charts.

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves [CASMPET-6721](https://jira-pro.it.hpe.com:8443/browse/CASMPET-6721)
* Change will also be needed in `istio`
* Future work required by [CASMPET-6960](https://jira-pro.it.hpe.com:8443/browse/CASMPET-6960)

## Testing

_List the environments in which these changes were tested._

### Tested on:

  * `starlord`
  * Virtual Shasta
  *  Local development environment(craystack VM)

### Test description:

Upgraded the istio charts to use the new 1.19.10 images 

- Were the install/upgrade-based validation checks/tests run (goss tests/install-validation doc)? Y
- Were continuous integration tests run? If not, why?
- Was upgrade tested? If not, why? Y
- Was downgrade tested? If not, why? Y
- Were new tests (or test issues/Jiras) created for this change?

## Risks and Mitigations

Low

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: kubernetes/cray-istio-deploy/values.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ kubectl:
```
Comment (dborman-hpe) [inline kubernetes/cray-istio-deploy/values.yaml:32]: You missed this one, can you also update this to `1.24.17`?

Comment (arka-pramanik-hpe) [reply kubernetes/cray-istio-deploy/values.yaml:32]: Rectified

Comment (dborman-hpe) [reply kubernetes/cray-istio-deploy/values.yaml:32]: Thanks!

**Review by dborman-hpe — APPROVED** [review]
LGTM

**Review by rajeshranganathan-hpe — APPROVED** [review]

**Review by KJ-HareshKumar — APPROVED** [review]

File: kubernetes/cray-istio/templates/ingress-gateway/deployment.yaml [hunk 2]
```diff
@@ -44,9 +47,13 @@ spec:
...
         service.istio.io/canonical-revision: latest
         {{- end }}
+        istio.io/rev: {{ $.Values.revision | default "default" }}
+        install.operator.istio.io/owning-resource: {{ $.Values.ownerName | default "unknown" }}
+        operator.istio.io/component: "IngressGateways"
       annotations:
+        istio.io/rev: {{ $.Values.revision | default "default" }}
         {{- if $.Values.global.meshConfig.enablePrometheusMerge }}
-        prometheus.io/port: "15090"
+        prometheus.io/port: "15020"
```
[the remark this answers is not in the document]
Comment (arka-pramanik-hpe) [reply kubernetes/cray-istio/templates/ingress-gateway/deployment.yaml:56]: The reason was [this](https://github.com/istio/istio/pull/26885)

You can refer to the Open Source PR which I have referred to in the [commit](https://github.com/Cray-HPE/cray-istio/commit/886cc36e2a95d346ee1d932f4561c299fbb17882) message for any discrepancy for each and every commit as well.

**Pull request merged**